bench-tps/src/bench.rs

```diff
@@ -713,13 +721,19 @@ fn compute_and_report_stats(
         );
     }
 
+    let total_tx_send_count = total_tx_send_count as u64;
+    let num_dropped = if total_tx_send_count > max_tx_count {
+        total_tx_send_count - max_tx_count
```

Comment:
We can do ```total_tx_send_count.saturated_sub(max_tx_count)```

Reply:
good call. I'm not sure I want to merge this though if it hides a bug in the fullnode code.